### PR TITLE
[Widget enhancement] 15 - Misc address comments

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

This PR addresses comments in PRs `01-14`
* [x] https://github.com/cowprotocol/cowswap/pull/3813#discussion_r1484509780
* [x] https://github.com/cowprotocol/cowswap/pull/3813#discussion_r1484519468
* [ ] https://github.com/cowprotocol/cowswap/pull/3815#discussion_r1484760030 
* [x] https://github.com/cowprotocol/cowswap/pull/3815#discussion_r1484526409
* [x] https://github.com/cowprotocol/cowswap/pull/3817#discussion_r1484539944
* [x] https://github.com/cowprotocol/cowswap/pull/3818#discussion_r1484559209
* [x] https://github.com/cowprotocol/cowswap/pull/3819#discussion_r1484587995
* [x] https://github.com/cowprotocol/cowswap/pull/3819#discussion_r1484818277
* [x] https://github.com/cowprotocol/cowswap/pull/3819#discussion_r1484583796
* [x] https://github.com/cowprotocol/cowswap/pull/3820#discussion_r1484823944
* [x] https://github.com/cowprotocol/cowswap/pull/3821#discussion_r1484827230
* [x] https://github.com/cowprotocol/cowswap/pull/3822#discussion_r1484836684
* [x] https://github.com/cowprotocol/cowswap/pull/3813#discussion_r1486139385

### CODE REVIEW

File: libs/events/src/types/events.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import {
+  OnExecutedOrderPayload,
+  OnPostedOrderPayload,
+  OnRejectedOrderPayload as OnCancelledOrderPayload,
+  OnPostedEthFlowOrderPayload,
+} from './orders'
+import { OnToastMessagePayload } from './toastMessages'
+
+// export type CowEvents = 'updateWidgetConfig' | 'onToastMessage' | 'onPostedOrder' | 'onExecutedOrder'
```
Comment (alfetopito) [inline libs/events/src/types/events.ts:9]: Dead code?

```suggestion
```

File: libs/events/src/types/orders.ts
Comment (alfetopito) [inline libs/events/src/types/orders.ts:1]: I'm just thinking: could we use these events to migrate away from redux middleware in the app?

**Review by alfetopito — APPROVED** [review]
Did you mean to point this to develop instead of `14` in the series?

File: libs/events/src/types/orders.ts
Comment (anxolin) [reply libs/events/src/types/orders.ts:1]: Sure, we could. I think could help us out to decouple things a bit more

**Pull request merged**